publications-2019.bib

```diff
@@ -312,21 +313,27 @@ @article{Wick2019OptimizationWN
   url={https://arxiv.org/abs/1910.03424}
 }
 
-@Article{NW19,
-  author =       {N. Noii and T. Wick},
-  title =        {A phase-field description for pressurized and non-isothermal
-                  propagating fractures},
-  note =      {submitted},
-  year =         {2019},
-  url =       {https://arxiv.org/abs/1904.00196}
+@article{Noii_2019,
+   title={A phase-field description for pressurized and non-isothermal propagating fractures},
+   volume={351},
+   ISSN={0045-7825},
+   url={http://dx.doi.org/10.1016/j.cma.2019.03.058},
+   DOI={10.1016/j.cma.2019.03.058},
+   journal={Computer Methods in Applied Mechanics and Engineering},
+   publisher={Elsevier BV},
+   author={Noii, Nima and Wick, Thomas},
+   year={2019},
+   month={Jul},
+   pages={860--890}
 }
 
-@Article{AE19,
-  author =       {J. C. Araujo Cabarcas and C. Engstr{\"o}m},
-  title =        {Removal of spurious solutions encountered in Helmholtz scattering resonance computations in $R^d$},
-  note =      {submitted},
-  year =         {2019},
-  url =       {https://arxiv.org/abs/1904.08812}
+@misc{cabarcas2019spurious,
+    title={On spurious solutions encountered in Helmholtz scattering resonance computations in R^d with applications to nano-photonics and acoustics},
```

Comment:
You need to escape the R^d

Reply:
Oof. I spent half an hour on trying to find this problem. The CI output wasn't useful, nor were my local tests -- they complained about a unicode character, which turned out to not actually exist...